packages/components/src/components/buttons/Button/Button.tsx

```diff
@@ -26,6 +27,7 @@ type ButtonContainerProps = TransientProps<AllowedFrameProps> & {
     $hasIcon?: boolean;
     $isFullWidth?: boolean;
     $isSubtle: boolean;
+    as?: 'a' | 'button';
```

Comment:
What's the reason for this? Isn't it already decided by the presence of a href prop?

Reply:
We need to change tag to `a` when we want to use `href`. it's not working on `button` AFAIK

Reply:
Yeah but I don't see you use it anywhere? Shouldn't the styled ButtonContainer change to something else than button?

Reply:
Solved in DM. it's native feature of styled components